casper/contracts/simple_casper.v.py

```diff
@@ -168,6 +174,13 @@ def get_deposit_size(validator_index: num) -> num(wei):
 def get_total_curdyn_deposits() -> wei_value:
     return floor(self.total_curdyn_deposits * self.deposit_scale_factor[self.current_epoch])
 
+
+@public
+@constant
+def get_total_destroyed() -> wei_value:
```

Comment:
No need to create helper function if you just declare `total_destroyed` above as `public(wei_value)`